cogboard-webapp/src/components/helpers/index.js

```diff
@@ -40,3 +12,7 @@ export const sortByKey = (obj, key, asc = true) => Object.entries(obj)
 
     return newObj;
   }, {});
+
+Array.prototype.insert = function ( index, item ) {
```

Comment:
Adding methods to build in objects prototypes is considered a bad idea. These are reserved for internal language features. Please create an util function.